### ISSUE TITLE
Can't create redirects, not taking Site into account

### ISSUE THREAD

**Issue by tomjrjones** [Dolphiq/craft3-plugin-redirect#98]
When a url is hit with a page that doesn't it exist it is caught in the `Registered catch all urls` section.

I then go to that particular entry and update the destination url which should then add a redirect entry.  However it is adding the entry under the wrong site and removing the starting `/`.

I thought I could just add a new redirect manually for the time being but even that is is adding the entry under the wrong site and removing the starting `/`.

Basically it does not seem to be taking into account the Site, which means now I can't create any redirect entries what so ever

**Comment by OwenMelbz** [comment]
Having same problem, abandoning the plugin for the time being :(
